packages/ui-components/src/components/kv-breadcrumb-list/test/kv-breadcrumb-list.e2e.ts

```diff
@@ -0,0 +1,72 @@
+import { E2EElement, E2EPage, newE2EPage } from '@stencil/core/testing';
+
+describe('KvBreadcrumbList (end-to-end)', () => {
+	let page: E2EPage;
+	let listEl: E2EElement;
+
+	afterAll(async () => {
+		await page.close();
+	});
+
+	describe('when the component has no children', () => {
+		beforeEach(async () => {
+			page = await newE2EPage();
+			await page.setContent(`<kv-breadcrumb-list></kv-breadcrumb-list>`);
+			listEl = await page.find('kv-breadcrumb-list');
+		});
+
+		it('should render the component', () => {
+			expect(listEl).toEqualHtml(`
```

Comment:
I don't think you should make assertions about HTML. This would be very error-prone for future changes.

Please consider making less precise assertions, for example on the number o `kv-breadcrumb-item` found here.